adversary/withhold.go

```diff
@@ -52,66 +52,71 @@ func (w *WithholdCommit) ReceiveAlarm(_ string) error {
 	return nil
 }
 
+func broadcastHelper(host sim.AdversaryHost, sender f3.ActorID) func(f3.Payload, *f3.Justification) {
```

Comment:
Nit: please move helpers to bottom of the file so more semantically interesting code comes first.

Also the tiny variable names do my head in.

Reply:
Yeah, the variable names should be a bit longer, the function grew up larger than I was planning.